AutoscoperM/AutoscoperM.py

```diff
@@ -768,26 +769,19 @@ def onLoadPV(self):
                 )
                 return
 
+            # load the combined scale and translation transform for each generated partial volume
             for i in range(len(vols)):
                 nodeName = os.path.splitext(os.path.basename(vols[i]))[0]
                 volumeNode = slicer.util.loadVolume(vols[i])
-                translationTransformFileName = os.path.join(mainOutputDir, transformSubDir, f"{nodeName}_t.tfm")
-                scaleTranformFileName = os.path.join(mainOutputDir, transformSubDir, f"{nodeName}_scale.tfm")
-                if not os.path.exists(translationTransformFileName):
+                transformFileName = os.path.join(mainOutputDir, transformSubDir, f"{nodeName}.tfm")
```

Comment:
I suggest to keep the check ensuring that `{nodeName}_scale.tfm` and `{nodeName}_t.tfm`

```python
                translationTransformFileName = os.path.join(mainOutputDir, transformSubDir, f"{nodeName}_t.tfm")
                if not os.path.exists(translationTransformFileName):
                    raise ValueError(
                        f"Failed to load partial volume {nodeName}: "
                        "Corresponding translation transform file {translationTransformFileName} not found"
                    )

                scaleTranformFileName = os.path.join(mainOutputDir, transformSubDir, f"{nodeName}_scale.tfm")
                if not os.path.exists(scaleTranformFileName):
                    raise ValueError(
                        f"Failed to load partial volume {nodeName}: "
                        "Corresponding scaling transform file {scaleTranformFileName} not found"
                    )
```